pkg/driver_atwinc15x0/Kconfig

```diff
@@ -8,12 +8,7 @@
 config PACKAGE_DRIVER_ATWINC15X0
     bool "ATWINC15x0 WiFi driver package"
     depends on TEST_KCONFIG
-    depends on HAS_PERIPH_SPI
-    depends on HAS_PERIPH_GPIO
-    depends on HAS_PERIPH_GPIO_IRQ
-    select MODULE_PERIPH_SPI
-    select MODULE_PERIPH_GPIO
-    select MODULE_PERIPH_GPIO_IRQ
+    depends on MODULE_ATWINC15X0
```

Comment:
Is this also like with the bme680 package? If this is not really usable without the driver, let us just remove the prompt as well.

Reply:
Yes, it's the exactly the same as with BME680. In fact, I stumbled with the dependencies in the BME680 driver when I was looking for how to implement Kconfig for drivers that depend on a package.

Reply:
Fixed with 63703a02e2d4b11a7a731b7cb329cf95d443d67a.